core/controllers/topic_editor_test.py

```diff
@@ -95,13 +98,13 @@ def test_get(self):
 
         with self.swap(constants, 'ENABLE_NEW_STRUCTURE_EDITORS', True):
             self.login(self.ADMIN_EMAIL)
-            with self.swap(constants, 'NUM_QUESTIONS_PER_PAGE', 1):
+            with self.swap(constants, 'NUM_QUESTION_SKILL_LINKS_PER_PAGE', 3):
```

Comment:
I'm kind of confused -- I understood NUM_QUESTIONS_PER_PAGE because the topic editor would have several pages of questions and it might display, say, 20 questions per page. But what does "NUM_QUESTION_SKILL_LINKS_PER_PAGE" mean? I don't think I follow the intuition...

Reply:
Like what I said in my last comment, we'll have to increase the num because there are multiple skills linked to one question and the backend can only get one question_skill_link each time. And to solve this, I fetch more question_skill_links than the number of questions.